msteams-platform/concepts/deploy-and-publish/overview.md

```diff
@@ -1,56 +1,72 @@
+AppSource provides a convenient location from which you can distribute your apps and other Office 365 extensibility types, such as Office add-ins and SharePoint add-ins. For more information on how to publish your app to AppSource, see [submit your app to AppSource](../appsource/publish.md).
 
-## Government Community Cloud (GCC) organizations
+## Distribute to Government Community Cloud (GCC) organizations
 
-### Upload your custom app directly to Teams
+A government organization that fulfills specific compliance requirements is a member of the GCC. The decision to upload an app or to publish an app is taken by the GCC tenant administrator. To upload the apps to your organization environment or publish the apps to your organization app catalog follow the steps in [uploading your app directly to Teams](#upload-your-app-directly-to-teams).
 
- As a GCC tenant administrator, you'll decide whether to upload a custom app to your tenant environment and whether to  publish it to your tenant app catalog. Microsoft doesn't own or control your custom applications, therefore, you must ensure that all endpoints are compliant with your organization's requirements. Additionally, if the app solution includes a bot or message extension, you'll need to complete the [Bot Framework](https://dev.botframework.com/) registration as follows:
+### Upload your app directly to Teams
+Microsoft does not own or control your custom applications, therefore, you must ensure that all endpoints are compliant with your organization's requirements. In addition, if the app solution includes a bot or message extension, you must complete the [Bot Framework registration](https://dev.botframework.com/).
 
-1. On the **Connect to channels** page, under **Add a featured channel**, select **Teams**.
-1. Navigate to the **Configure MSTeams** page (*see* below).
-1. Under **Messaging** select the **Microsoft Teams for Government Customers** radio button.
-1. In the lower left corner of the page, select **Save**.  
-
->[!IMPORTANT]
-> You cannot use the Teams commercial configuration to upload/sideload your custom app to a GCC environment,  You must select the **Microsoft Teams for Government Customers** radio button for a GCC compliant configuration.
+**To upload an app to your organization's environment**
 
+1. On the **Connect to channels** page, under **Add a featured channel**, select **Teams**.
+2. Navigate to the **Configure MSTeams** page.
+3. Under **Messaging**, select *Microsoft Teams for Government*.
 ![Teams messaging configuration page](../../assets/images/gcc-configure.png)
+   > [!IMPORTANT]
+   > You cannot use the *Microsoft Teams Commercial* configuration to upload or sideload your custom app to a GCC environment.
+4. In the lower left corner of the page, select **Save**.
```

Comment:
I think this flow and other GCC flows need more clarity. I am not sure if this is talking about uploading for a user/team (sideloading) or for an organization. Also, it is not clear how I get to step 1 (the Bot Framework link takes me to a marketing page).